src/components/split-button/split-button-children.style.js

```diff
@@ -0,0 +1,71 @@
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import StyledButton from '../button/button.style';
+import colors from './split-button-colors.style';
+// import OptionsHelper from '../../utils/helpers/options-helper';
+
+const StyledSplitButtonChildrenContainer = styled.div`
+  max-width: 100%;
+  position: absolute;
+  right: 0;
+  ${styleChildrenContainer}
+  
+  ${StyledButton} {
+    ${applyStylingToChildButtons}
+    color: ${colors.white};
+    margin-left: 0;
+    margin-top: 3px;
+    min-width: 100%;
+    z-index: 10;
+    text-align: left;
+  }
+
+  ${({ displayButtons }) => (!displayButtons ? 'display: none' : undefined)}
+`;
+
+function styleChildrenContainer({ theme }) {
+  if (theme.name === 'classic') {
```

Comment:
what about importing themes and comparing with 'themes.classic' instead of comparing to a string?

Reply:
Yeah meant to update that, just slipped my mind